frontend/packages/kubevirt-plugin/src/components/vms/vm-details.tsx

```diff
@@ -93,11 +96,18 @@ export const VMDetails: React.FC<VMDetailsProps> = (props) => {
   const OSMismatchExists =
     vmi && guestAgentInfoRaw && isWindows(vmiLike) !== (operatingSystemID === 'mswindows');
   const OSMismatchAlert = OSMismatchExists && (
-    <Alert className="co-alert" variant="warning" title="Operating system mismatch" isInline>
-      The operating system defined for this virtual machine does not match what is being reported by
+    <Alert
+      className="co-alert"
+      variant="warning"
+      title={t('kubevirt-plugin~Operating system mismatch')}
+      isInline
+    >
+      {t(
+        `kubevirt-plugin~The operating system defined for this virtual machine does not match what is being reported by
```

Comment:
Don't use backticks inside the `t()` function. The i18n parser can't handle them for some reason and it won't pick up the key.

Reply:
yea, it's for multi line text, the i18n don't like backticks because it will not parse them, but here is just text,
@glekner do you think backticks are legit here (given they will not be parsed inside the `t` method) ? do you have a nicer solution for multi line translatable text ?

Reply:
changing to "" + "" ...
if anyone have a nicer way please comment.

Reply:
The `Trans` component looks nice there. Let's hope that there aren't any unforeseen side effects that make us remove it. =)